spec/models/news_search_spec.rb

```diff
@@ -14,16 +14,32 @@
   describe '#initialize(options)' do
     let(:feed) { affiliate.rss_feeds.first }
 
-    def filterable_search_options
-      { affiliate: affiliate, channel: feed.id }
+    let(:filterable_search_options) do
+      { affiliate: affiliate,
+        channel: feed.id }
     end
 
     it_behaves_like 'an initialized filterable search'
 
     context 'when options does not include sort_by' do
       subject(:search) { described_class.new filterable_search_options }
-      its(:sort_by_relevance?) { should be false }
-      its(:sort) { should eq('published_at:desc') }
+
+      its(:sort_by_relevance?) { is_expected.to be false }
+      its(:sort) { is_expected.to eq('published_at:desc') }
+    end
+
+    context 'when show_only_media_content is false' do
```

Comment:
We weren't currently testing this existing tag-setting behavior, so I added these in to clarify how NewsSearch behaves different from the other FilterableSearches (e.g. i14y and blended).